docs/custom_fields.md

```diff
@@ -2,134 +2,138 @@
 
 Presidio supports custom fields using either online via a simple REST API or by adding new PII recognizers in code. The underlying object behind each field is called a 'Recognizer'. This documentation describes how to add new recognizers by API or code.
 
-1. API
+## Customization options
+1. [Via API](#via-api)
+2. [Via code](#via-code)
 
-    a. Getting a recoginzer
+## Via Presidio's API <a name="via-api"></a>
 
-    ```sh
-    GET <api-service-address>/api/v1/analyzer/recognizers/<recognizer_name>
-    ```
+  a. Getting a recoginzer
 
-    b. Getting all recognizers
+  ```sh
+  GET <api-service-address>/api/v1/analyzer/recognizers/<recognizer_name>
+  ```
 
-    ```sh
-    GET <api-service-address>/api/v1/analyzer/recognizers
-    ```
+  b. Getting all recognizers
 
-    c. Creating a new recoginzer
+  ```sh
+  GET <api-service-address>/api/v1/analyzer/recognizers
+  ```
 
-    ```sh
-    POST <api-service-address>/api/v1/analyzer/recognizers/<new_recognizer_name>
-    ```
+  c. Creating a new recoginzer
 
-    ### Request structure:
+  ```sh
+  POST <api-service-address>/api/v1/analyzer/recognizers/<new_recognizer_name>
+  ```
```

Comment:
d) updating a recognizer is exactly the same as c) only its with PUT

Reply:
Added